app/diversity/get.controller.js

```diff
@@ -0,0 +1,24 @@
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../common/utils/getCommentText')
+
+const getDiversity = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planid }, session: { 'x-auth-token': token } },
+  res
+) => {
+  let renderDetails = renderInfo
+  if (!renderInfo) {
+    renderDetails = {}
+  }
+  renderDetails.backurl = path.substring(0, path.lastIndexOf('/'))
```

Comment:
If most pages need a back url, it might be worth having this in a express 'use' for all pages and extracting the origin from the header mebe?

Reply:
Agreed - will raise a ticket

Reply:
Ticket created as SP-479